command/connect/envoy/envoy.go

```diff
@@ -145,32 +160,28 @@ func (c *cmd) init() {
 	c.help = flags.Usage(help, c.flags)
 }
 
-const (
-	DefaultMeshGatewayPort int = 443
-)
-
 func parseAddress(addrStr string) (string, int, error) {
 	if addrStr == "" {
 		// defaulting the port to 443
-		return "", DefaultMeshGatewayPort, nil
+		return "", DefaultGatewayPort, nil
```

Comment:
Just wanted to note that @dnephin has a PR open that extracts all of this address parsing out. One of you will have some conflicts it would seem.

Reply:
Thanks for the heads up. I'll reach out to get his merged first and I'll resolve the conflict.